components/avatar/avatar.css

```diff
@@ -0,0 +1,44 @@
+.avatar-group{
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    margin: 1rem;  
+    max-width: 600px;
+    border: 1px solid rgb(223, 218, 218);
+}
+
+
+.avatar{
+  display: block;
+  object-fit: cover;
+  border-radius: 999px; 
+  width: 4rem;
+  height: 4rem;
+  margin: 1rem;
+}
+
```

Comment:
if you are using multiple sizes , please use 'avatar-xs' , instead of 'avatar xs' , as it is generic in the sense you will use similar classes for other components based on different sizes